src/hooks/Transactor.ts

```diff
@@ -4,22 +4,21 @@ import { Contract } from '@ethersproject/contracts'
 import { Deferrable } from '@ethersproject/properties'
 import { TransactionRequest } from '@ethersproject/providers'
 import { parseUnits } from '@ethersproject/units'
-import Notify, { InitOptions, TransactionEvent } from 'bnc-notify'
-import { ThemeContext } from 'contexts/themeContext'
-import { Signer } from 'ethers/lib/ethers'
+import { Signer, Transaction } from 'ethers/lib/ethers'
 import { useCallback, useContext } from 'react'
 
 import { emitErrorNotification } from 'utils/notifications'
 
 import { t } from '@lingui/macro'
 import * as Sentry from '@sentry/browser'
-import { windowOpen } from 'utils/windowUtils'
 
+import { TxHistoryContext } from 'contexts/txHistoryContext'
 import { useWallet } from './Wallet'
 
-type TransactorCallback = (e?: TransactionEvent, signer?: Signer) => void
+type TransactorCallback = (e?: Transaction, signer?: Signer) => void
 
 type TransactorOptions = {
+  title?: string
```

Comment:
this allows us to define a title for every TransactorInstance (i.e. "Pay <project name>", "Redeem $<token>", etc)

it will default to using the functionName. so far in this PR a title has only been defined for the pay tx